### PR TITLE
Refactor base image

### PR DESCRIPTION
**Pull request by joshpollara**

### CODE REVIEW

**Comment by joshpollara** [comment]
@orbitz I still need to test

File: bin/aws [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```
Comment (orbitz) [inline bin/aws:1]: Does this script download everytime  or only the first time?

**Review by orbitz — DISMISSED** [review]
Had a question on AWS script but other than that looks ok.

File: bin/aws (hunk 1 above)
Comment (joshpollara) [reply bin/aws:1]: Only the first time. The aws installer overwrites this file with a symlink automatically.

**Review by orbitz — APPROVED** [review]

**Pull request merged**